packages/wells/src/client/cogniteClient.ts

```diff
@@ -4,17 +4,30 @@ import { version } from '../../package.json';
 import { Wells } from './api/wells';
 import { Wellbores } from './api/wellbores';
 import { accessApi } from '@cognite/sdk-core';
+import { Surveys } from './api/surveys';
 
 export default class CogniteClient extends CogniteClientStable {
   private wellsSDK?: Wells;
   private wellboresSDK?: Wellbores;
+  private surveysSDK?: Surveys;
 
   protected initAPIs() {
     super.initAPIs();
 
     // Turns into $BASE_URL/api/$API_VERSION/projects/$PROJECT/assets
     this.wellsSDK = this.apiFactory(Wells, 'assets');
     this.wellboresSDK = this.apiFactory(Wellbores, 'assets');
+
+    // Turns into $BASE_URL/api/$API_VERSION/projects/$PROJECT/sequences
+    this.surveysSDK = this.apiFactory(Surveys, 'sequences');
+
+    /* We want to hide the dependency injection from the users
+     of the SDK. This allows surveys endpoints being accessed on the
+     wellbores object using 'this'. This also ensures
+     a Singleton pattern with only one CogniteClient being 
+     instanciated and we reuse that connection even when
+     surveys endpoints are called within the wellbores class */
+    this.wellboresSDK.surveysSdk = this.surveysSDK;
```

Comment:
We want to hide the dependency injection from the users of the SDK. This allows surveys endpoints being accessed on the wellbores object using 'this'. This also ensures a Singleton pattern with only one CogniteClient being instanciated and we reuse that connection even when surveys endpoints are called within the wellbores class.

added a public setter and private getter inside the wellbores class: 
![image](https://user-images.githubusercontent.com/25263842/98006392-78265000-1df2-11eb-90c4-25777bc049eb.png)

the lazy getter can now access the surveys endpoints of 'this' with the same client instance like this:
![image](https://user-images.githubusercontent.com/25263842/98006556-ae63cf80-1df2-11eb-99c3-d967d264a6bf.png)

I took some time to figure this one out, because I really didn't want to ingest the CogniteClient into each of the wellbores endpoints to be able to use surveys endpoints within the wellbores class.